implementations/rust/ockam/ockam/src/protocols/stream/requests.rs

```diff
@@ -2,65 +2,70 @@
 
 use crate::protocols::ProtocolPayload;
 use serde::{Deserialize, Serialize};
+use serde_bare::Uint;
 
 /// Request a new mailbox to be created
+#[repr(C)]
```

Comment:
Is that required for bare serialization to work correctly?

Reply:
Turns out no. I added those to debug a problem I was having, but it turned out to be unrelated. Thanks for pointing this out, I'll remove those again!